### PR TITLE
PrizesPage

### PR DESCRIPTION
**Pull request by EdgarAllan-Bro**

### CODE REVIEW

File: frontend/src/Pages/PrizesPage/PrizesPage.jsx
Comment (CooperW824) [inline frontend/src/Pages/PrizesPage/PrizesPage.jsx:1]: I personally am not a fan of styled-components, they can definitely be useful for some things, but sometimes they make CSS harder to read and understand. For your future reference I would avoid them unless you have JS interacting with the CSS. I would instead use classes and IDs in a CSS file that you import.

Comment (CooperW824) [reply frontend/src/Pages/PrizesPage/PrizesPage.jsx:1]: But this works and looks great! Just something to think about for future react projects. Great job though everyone!

File: frontend/src/Pages/PrizesPage/img/gear2_ps.png
Comment (CooperW824) [inline frontend/src/Pages/PrizesPage/img/gear2_ps.png:1]: I don't see this being used on the page can we remove it? Along with the other images that aren't being used?

File: frontend/src/Pages/PrizesPage/tab.jsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  border: none;
+  display: block;
+  min-height: fit-content;
+`;
+
+const Prize = styled.div`
+  padding: 8px 16px;
+  color: white;
+  white-space: normal;
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
```
Comment (CooperW824) [inline frontend/src/Pages/PrizesPage/tab.jsx:58]: Can we make this a sans-serif font like the Poppins Font like use on other pages on the website?

**Review by CooperW824 — CHANGES_REQUESTED** [review]
This looks really good, just some minor tweaks:
- Can you remove the unused images from the images folder
- Can you change the font on the prize section to be a font like the Poppins font we use on other pages?

File: frontend/src/Pages/PrizesPage/wireframe/sample1.jpg
Comment (maryellenmarino) [inline frontend/src/Pages/PrizesPage/wireframe/sample1.jpg:1]: can we delete the wireframe folder altogether? We have a design repo for these purposes and shouldn't have it go into the main build.

**Review by CooperW824 — APPROVED** [review]
LGTM! Great Job!

**Review by CooperW824 — APPROVED** [review]

**Pull request merged**